src/logic/solution/solution.ts

```diff
@@ -47,22 +47,22 @@ export class Market {
         {
             baseSymbol: CurrencySymbol.EUR,
             quoteSymbol: CurrencySymbol.USD,
-            bid: 1.182,
-            ask: 1.182,
+            bid: 1.1313,
+            ask: 1.1323,
         },
         {
-            baseSymbol: CurrencySymbol.GBP,
+            baseSymbol: CurrencySymbol.PLN,
```

Comment:
Small issue - there's no such pair as PLN/USD, its other way around (USD/PLN). If that's not too big of a change, you could revert it. Otherwise, we could go back to GBP